### PR TITLE
Issue #1986 User friendly error message for *.addon.remove file

### PR DESCRIPTION
**Pull request by amitkrout**
Fix : #1986

### CODE REVIEW

**Comment by amitkrout** [comment]
This failure is coming from file ```provision_earlier_version.feature``` in starting minishift, resulting some dependent steps skip and failure. The same feature file executed successfully in my local fedora27. So In re-triggering there is a great chance of not hitting the same failure issue. 

This king of failure need to be documented in the upstream as ```Integration test most common failure``` IMO. It would be really helpful for the new contributor mostly newbie for example if the pr fails with these errors they should not be in panic. @minishift/minishift-dev Thoughts ?

**Comment by LalatenduMohanty** [comment]
retest this please

File: cmd/minishift/cmd/addon/addon.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ const (
 
 	emptyAddOnError      = "You must specify an add-on name. Run 'minishift addons list' to view installed add-ons."
 	noAddOnMessage       = "No add-on with the name '%s' is installed."
-	noRemoveAddOnMessage = "No %s.addon.remove file found for '%s' add-on."
+	noRemoveAddOnMessage = "Unable to remove addon: '%s'. No %s.addon.remove file is found."
```
Comment (LalatenduMohanty) [inline cmd/minishift/cmd/addon/addon.go]: s/addon: '%s'/e addon '%s'/. We should remove the `:`.

**Comment by LalatenduMohanty** [comment]
> This king of failure need to be documented in the upstream as Integration test most common failure IMO. It would be really helpful for the new contributor mostly newbie for example if the pr fails with these errors they should not be in the panic. @minishift/minishift-dev Thoughts ?

Actually, I have not seen this failure much in the past. Not sure if others have seen it. IMO we can not take any test failure as known issue or common failure. So we should do something to the fix the test itself and not be documenting this as known issues.

**Comment by amitkrout** [comment]
> Actually, I have not seen this failure much in the past. Not sure if others have seen it. 

This is not consistent in ci even local system i seldom face such failure. 
 
> IMO we can not take any test failure as known issue or common failure. So we should do something to the fix the test itself and not be documenting this as known issues.

Sure, in the next appearance of such failure, we will file a github issue.

File: cmd/minishift/cmd/addon/addon.go (hunk 1 above)
Comment (amitkrout) [reply cmd/minishift/cmd/addon/addon.go]: Done

**Pull request merged**